src/api/apiUtils.js

```diff
@@ -1,9 +1,11 @@
+import { camelizeKeys, decamelizeKeys } from 'humps';
+
 const baseUrl = process.env.REACT_APP_API_URL;
 
 export async function handleResponse(response) {
-  if (response.ok) return response.json();
+  if (response.ok) return camelizeKeys(await response.json());
   if (response.status >= 400 && response.status < 500) {
-    const error = await response.json();
+    const error = camelizeKeys(await response.json());
     throw error;
   }
   throw new Error('Network response was not ok.');
```

Comment:
I think you should add this logic inside a try and catch block in case the `await response.json()` returns an error. I think something like this could work:  
```
export async function handleResponse(response) {
if (response) {
 try {
  const json = await response.json(); 
  const camelizedResponse = camelizeKeys(json);
  if (response.ok) return camelizedResponse;
  if (response.status >= 400 && response.status < 500) throw camelizedResponse; 
 } catch (error) {
  throw error
  }
}
 throw new Error('Network response was not ok.');
};
```

Reply:
Nice suggestion! I like it!
I did the change but I didn't include the `try {} catch {}` because prettier said that it was redundant